geopandas/sindex.py

```diff
@@ -225,11 +239,16 @@ def is_empty(self):
 
     from . import geoseries  # noqa
     from .array import GeometryArray  # noqa
-    from pygeos import STRtree, box, points, Geometry  # noqa
+    from pygeos import STRtree, box, points, Geometry, from_shapely, from_wkb  # noqa
```

Comment:
Can you do `import pygeos`, and then `pygeos.from_shapel(..)` etc? Since we also have `from_shapely` and `from_wkb` functions in geopandas, this gets confusing in the code to know which one it is

Reply:
Sure!